### PR TITLE
4.6 Release Notes for Image Registry

### PR DESCRIPTION
**Pull request by bmcelvee**
Replaces https://github.com/openshift/openshift-docs/pull/26422.

### CODE REVIEW

**Comment by vikram-redhat** [comment]
good to merge after peer review.

**Review by vikram-redhat — DISMISSED** [review]

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -822,6 +822,61 @@ The Local Storage Operator now has the ability to:
...
+==== Change the image pruner's log level
+
+Cluster administrators can now configure `logLevel` in the Pruning Custom Resource to debug logs.
+
+//See xre:../applications/pruning-objects.adoc#pruning-images_pruning-objects[Automatically pruning images] for more information.
+
+[id="ocp-4-6-registry-azure-gov"]
+==== Image Registry supports Azure Government
+
+You can now set up and configure the Image Registry for Azure Government.
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:843]: Does `Image Registry` need to be capitalized? I noticed it is mostly lowercase in the docs unless referring to the Operator.

**Review by codyhoag — DISMISSED** [review]
One question; otherwise LGTM!

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (bmcelvee) [reply release_notes/ocp-4-6-release-notes.adoc]: Thank you - I was in Operator mindset. Updating now!

**Comment by bmcelvee** [comment]
Crumbs - please hold on re-approving.

**Comment by bmcelvee** [comment]
Ok all set for re-approving!

**Review by codyhoag — APPROVED** [review]

**Pull request merged**